gamemodes/terrortown/gamemode/scoring.lua

```diff
@@ -232,7 +232,7 @@ function SCORE:StreamToClients()
     end
 
     events = util.Compress(events)
-    if events == "" then
+    if #events == 0 then
```

Comment:
`util.Compress` returns `nil` if the input string is 0-length. I don't think that can happen here, but just in case
```suggestion
    if events == nil or #events == 0 then
```